dispatcher/src/conf.d/available_vhosts/wknd.vhost

```diff
@@ -60,5 +58,20 @@ Include conf.d/variables/custom.vars
     ErrorDocument 502 ${500_PAGE}
     ErrorDocument 503 ${500_PAGE}
     ErrorDocument 504 ${500_PAGE}
-
+    # AEM Theme Sources: cache mutable (but very likely not changed) resources for 24h on CDN and background refresh to avoid MISS on CDN
+    <LocationMatch "^/etc\.clientlibs/.*\.(?i:json|png|gif|jpe?g)$">
+        Header set Cache-Control "max-age=7200,s-maxage=86400,stale-while-revalidate=43200,stale-if-error=43200,public"
+    </LocationMatch>
+    # AEM Theme Sources: cache immutable resources for 1year (as recommended by Lighthouse), background refresh to avoid MISS on CDN
+    <LocationMatch "^/etc\.clientlibs/.*\.(?i:js|css|ttf|svg)$">
+        Header set Cache-Control "max-age=31536000,stale-while-revalidate=43200,stale-if-error=43200,public,immutable"
+    </LocationMatch>
+    # HTML pages: CDN cache for 10min with background refresh to avoid MISS on CDN
+    <LocationMatch "\.html$">
+        Header set Cache-Control "max-age=60,s-maxage=600,stale-while-revalidate=43200,stale-if-error=43200"
+    </LocationMatch>
+    # Core Component Image Component has immutable URLs, cache for 1year (as recommended by Lighthouse), background refresh to avoid MISS on CDN
+    <LocationMatch "\.coreimg\..*\.(?i:jpe?g|png|gif|svg)$">
+        Header set Cache-Control "max-age=31536000,stale-while-revalidate=43200,stale-if-error=43200,public,immutable"
+    </LocationMatch>
```

Comment:
@keepthebyte these seem generic (not specific to WKND). Should this also be committed as part of the `default.vhost` in the archetype? https://github.com/adobe/aem-project-archetype/blob/master/src/main/archetype/dispatcher.cloud/src/conf.d/available_vhosts/default.vhost

Reply:
I think I want to apply that to a few more sites 1st.. - I'm planning to work with a few customers and other internal sites - after that I think they will be more proven - and we can discuss making them default for AEM impl..